packages/moonstone/Input/InputSpotlightDecorator.js

```diff
@@ -27,6 +27,24 @@ const handleKeyDown = handle(
 	call('onKeyDown')
 );
 
+
+/**
+ * Default config for {@link moonstone/Input.InputSpotlightDecorator}
+ * @hocconfig
+ * @memberof moonstone/Input.InputSpotlightDecorator
```

Comment:
Same above.

Reply:
I checked it comparing to other components. I think it looks fine the way it is.